app/views/devise/registrations/edit.slim

```diff
@@ -0,0 +1,45 @@
+.container-fluid
+  .auth
+    h2 Edit
+
+    = simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
+      = f.error_notification
+
+      .form-inputs
+        = f.input :first_name, required: true, input_html: {autocomplete: "name"}, placeholder: "#{}"
```

Comment:
remove please autocomplete here and in all places below

Reply:
placeholder should be more informative. For example `First name` or `Enter your first name`